### PR TITLE
fix: [L02]: Duplicate function execution in commitAndEmitEncryptedVote

### PR DESCRIPTION
**Pull request by chrismaree**
**Motivation**

*OZ identified the following issue:*

```
In the VotingV2 contract the overloaded function commitAndEmitEncryptedVote that is
provided for backwards compatibility mistakenly calls commitVote twice: The first call to
commitVote is performed directly within the function body on line 614, and an additional call
is performed indirectly by the call to commitAndEmitEncryptedVote on line 616.
The duplicate call is an unnecessary gas expense and leads to an unintentional duplication of
VoteCommitted event emission.
```

*Fix presented in this PR:*

Remove the duplicate call to `commitVote`.

**Testing**

Check a box to describe how you tested these changes and list the steps for reviewers to test.

- [ ]  Ran end-to-end test, running the code as in production
- [ ]  New unit tests created
- [ ]  Existing tests adequate, no new tests required
- [X]  All existing tests pass
- [ ]  Untested

### CODE REVIEW

File: packages/core/contracts/oracle/implementation/Staker.sol [hunk 1]
```diff
@@ -307,13 +308,9 @@ contract Staker is StakerInterface, Ownable {
...
     // Determine if we are in an active reveal phase. This function should be overridden by the child contract.
-    function inActiveReveal() internal virtual returns (bool) {
-        return false;
-    }
+    function inActiveReveal() internal view virtual returns (bool) {}
 
-    function getStartingIndexForStaker() internal virtual returns (uint64) {
-        return 0;
-    }
+    function getStartingIndexForStaker() internal view virtual returns (uint64) {}
```
Comment (mrice32) [inline packages/core/contracts/oracle/implementation/Staker.sol:313]: nit: I think changes from https://github.com/UMAprotocol/protocol/pull/4068 were accidentally included. We should probably drop this commit.

File: packages/core/contracts/oracle/implementation/VotingV2.sol [hunk 2]
```diff
@@ -611,8 +611,6 @@ contract VotingV2 is
         bytes32 hash,
         bytes memory encryptedVote
     ) public override {
-        commitVote(identifier, time, "", hash);
```
Comment (mrice32) [inline packages/core/contracts/oracle/implementation/VotingV2.sol:614]: Wow, this is a great catch!

**Review by mrice32 — COMMENTED** [review]
LGTM once extra changes are removed!

**Review by mrice32 — APPROVED** [review]

**Review by md0x — APPROVED** [review]
Nice catch. LGTM

**Pull request merged**